CONTRIBUTING.md

```diff
@@ -1,16 +1,239 @@
+[feature-requests]: #feature-requests
 
-6. Use [Labels](https://github.com/RIOT-OS/RIOT/wiki/Labels) to help classify pull requests and issues.
+Before opening a new feature request, check the
+[existing feature requests](https://github.com/RIOT-OS/RIOT/issues?q=state:open+type:issue+label:"Feature+Request")
+if there's one already opened on the same topic.
 
-7. Please adhere to our [code of conduct](https://github.com/RIOT-OS/RIOT/blob/master/CODE_OF_CONDUCT.md)
+To request new features or enhancements, just open a new
+[feature request issue](https://github.com/RIOT-OS/RIOT/issues/new?template=feature_request.md).
+Describe your needs and use cases if any. Give detail on why this is important
+and/or necessary for RIOT.
+
+## Bug Reports
+[bug-reports]: #bug-reports
+
+While bugs are unfortunate, they're a reality in software. We can't fix what we
+don't know about, so please report liberally. If you're not sure if something
+is a bug or not, feel free to file a bug anyway.
+
+**If you believe reporting your bug publicly represents a security risk to
+RIOT users, please send an email describing the bug to security@riot-os.org**.
+
+Before reporting a bug, have a look at
+[opened bugs](https://github.com/RIOT-OS/RIOT/issues?q=state:open+type:issue+label:bug).
+Maybe someone has already reported your error.
+
+Once you have verified this, opening an issue is as easy as clicking on [this
+link](https://github.com/RIOT-OS/RIOT/issues/new?template=bug_report.md) and
+filling out the fields.
+
+Each bug report issue uses a template with 5 sections that are here to help
+maintainer understand your issue and eventually reproduce it:
+
+    #### Description
+
+    #### Steps to reproduce the issue
+
+    #### Expected results
+
+    #### Actual results
+
+    #### Versions
+
+To fill the `version` section, use the script provided in the RIOT git
+repository:
+```
+./dist/tools/ci/print_toolchain_versions.sh
+```
+
+In summary, try to put as much information as possible that you think could
+help maintainers or other developers to fix the bug quickly.
+
+## Pull Requests
+[pull-requests]: #pull-requests
+
+Pull Requests (PR) are the primary mechanism used to change RIOT. GitHub itself
+has some great documentation on
+[using the Pull Request feature][about-pull-requests].
+We use the [fork and pull model][development-models], where contributors push
+changes to their personal fork and create pull requests to bring those changes
+into the source repository.
+
+[about-pull-requests]: https://help.github.com/articles/about-pull-requests/
+[development-models]: https://help.github.com/articles/creating-a-pull-request-from-a-fork
+
+* [General rules](#general-rules)
+* [Git usage](#git-usage)
+
+### General rules
+[general-rules]: #general-rules
+
+* Before opening a new Pull Request, have a look at
+  [existing ones](https://github.com/RIOT-OS/RIOT/pulls). Maybe someone has
+  already opened one about the same thing and if it's the case, you might be
```

Comment:
s/about the same thing and if it's the case/about the same thing. If that's the case